bundle/edu.gemini.spModel.core/src/main/java/edu/gemini/spModel/core/ProgramTypeEnum.java

```diff
@@ -0,0 +1,18 @@
+package edu.gemini.spModel.core;
+
+/**
+ * A program type enum companion to the Scala ProgramType, meant to facilitate
+ * use from Java.
+ */
+public enum ProgramTypeEnum {
+    C,
+    CAL,
+    DD,
+    DS,
+    ENG,
+    FT,
+    LP,
+    Q,
+    SV
+    ;
+}
```

Comment:
This may seem like a maintenance issue, since it has to parallel the Scala `ProgramType`.  `ProgramType` has a `typeEnum` method to return the corresponding enum element.  If we add a new `ProgramType` we also have to implement `typeEnum` for it in order for it to compile.

Reply:
At least that will force us to remember the maintenance issue if we ever need to add more `ProgramType`s.